packages/nc-gui/components/smartsheet/Form.vue

```diff
@@ -422,27 +422,27 @@ watch(view, (nextView) => {
           </div>
 
           <div class="flex flex-wrap gap-2 mb-4">
-            <button
+            <NcButton
               v-if="hiddenColumns.length"
-              type="button"
-              class="nc-form-add-all color-transition bg-white transform hover:(text-primary ring ring-accent ring-opacity-100) active:translate-y-[1px] px-2 py-1 shadow-md rounded"
+              type="secondary"
```

Comment:
No such `secondary` type for NcButton.

```
'link' | 'default' | 'primary' | 'ghost' | 'dashed' | 'text' | 'danger' | undefined
```

Reply:
```css
.nc-button.ant-btn-secondary {
  @apply bg-white border-1 border-gray-200 text-gray-700 md:(hover:bg-gray-100);
}
``` 

there is  secondary class in ant and we are over riding  it

Reply:
From ant design webpage, it seems that `secondary` is `default`. I guess we have default, primary, and secondary. Anyway, adding `secondary` to Button props is fine to me.